src/type/complex/function/complex.js

```diff
@@ -78,11 +78,11 @@ function factory (type, config, load, typed) {
         return new type.Complex(x.re, x.im)
       }
 
-      if ('r' in x && 'phi' in x) {
+      if (('r' in x && 'phi' in x) || ('abs' in x && 'arg' in x)) {
         return new type.Complex(x)
       }
 
-      throw new Error('Expected object with either properties re and im, or properties r and phi.')
+      throw new Error('Expected object with either properties re and im, or properties r/abs and phi/arg.')
```

Comment:
Sorry @ssakat, I meant to flag this up the first time. I obviously forgot to press enter or something.

The slashes are slightly confusing. I think this would be clearer:
`throw new Error('Expected object with either properties re and im, r and phi, or abs and arg.')`

Reply:
@harrysarson - no worries. Is this fine - `throw new Error('Expected object with properties (re and im) or (r and phi) or (abs and arg)')`?  I removed either as we have 3 options now.

Reply:
Looks great :)